coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -461,16 +461,16 @@ def set_segment_messages_metrics(self, segment_id, messages_metrics):
         """
         self.get_segment_messages_metrics_ref(segment_id).set(messages_metrics.to_firebase_map())
 
-    def compute_segment_coding_progress(self, segment_id, messages=None):
+    def compute_segment_messages_metrics(self, segment_id, messages=None):
```

Comment:
I still think this is cleaner if it only takes messages, but not a priority to change

Reply:
Noted, I'll update later. We still need the segment id to get all the code schemes.